projects/README.md

```diff
@@ -2,14 +2,14 @@
 
 |project|license|description|version|repository|patch/diff|
 |-------|-------|-----------|-------|----------|----------|
-|[cppzmq](https://github.com/zeromq/cppzmq 'cppzmq')|[MIT](https://github.com/zeromq/cppzmq/blob/v4.2.1/LICENSE 'MIT')|cppzmq is a minimal c++ binding to the libzmq functions.|4.2.1|[repo](https://github.com/zeromq/cppzmq/tree/v4.2.1 'cppzmq repo on github')|none|
+|[cppzmq](https://github.com/zeromq/cppzmq 'cppzmq')|[MIT](https://github.com/zeromq/cppzmq/blob/v4.2.2/LICENSE 'MIT')|cppzmq is a minimal c++ binding to the libzmq functions.|4.2.2|[repo](https://github.com/zeromq/cppzmq/tree/v4.2.2 'cppzmq repo on github')|none|
 |[GLEW](http://glew.sourceforge.net/ 'The OpenGL Extension Wrangler Library')|[open](http://glew.sourceforge.net/credits.html 'Modified BSD, Mesa 3-D (MIT), and Khronos (MIT)')|The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.|1.13.0|[repo](https://github.com/nigels-com/glew 'GLEW repo on github')|none|
-|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[patch](../patches/hdf5.patch 'patch')|
+|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.22|none|[patch](../patches/hdf5.patch 'patch')|
```

Comment:
This didn't actually get updated did it?  Should revert the revision number back to 1.8.16.